app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -5,22 +5,19 @@ EAPI=5
 CROS_WORKON_PROJECT="coreos/rkt"
 CROS_WORKON_LOCALNAME="rkt"
 CROS_WORKON_REPO="git://github.com"
-inherit cros-workon systemd
+AUTOTOOLS_AUTORECONF=yes
+AUTOTOOLS_IN_SOURCE_BUILD=yes
+inherit autotools-utils cros-workon flag-o-matic systemd toolchain-funcs
 
 if [[ "${PV}" == 9999 ]]; then
     KEYWORDS="~amd64"
 else
-    CROS_WORKON_COMMIT="40ced98c320c056e343fe9c3eaeb90a4ff248936" # v0.5.5
+    CROS_WORKON_COMMIT="9579f4bf57851a1a326c81ec2ab0ed2fdfab8d24" # v0.7.0
     KEYWORDS="amd64"
 fi
 
-# Must be in sync with stage1/rootfs/usr_from_coreos/cache.sh
```

Comment:
This file does not exist anymore with rkt 0.7.7. It is replaced by [stage1/usr_from_coreos/usr_from_coreos.mk](https://github.com/coreos/rkt/blob/master/stage1/usr_from_coreos/usr_from_coreos.mk#L2):
```
UFC_IMG_RELEASE := "709.0.0"
UFC_IMG_URL := "http://alpha.release.core-os.net/amd64-usr/$(UFC_IMG_RELEASE)/coreos_production_pxe_image.cpio.gz"
```
This CoreOS image and the signature will be downloaded if `./configure --with-stage1=coreos` is used.